src/libraries/System.Security.Permissions/src/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,36 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.DirectoryServices.DirectoryServicesPermission</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net462/System.Security.Permissions.dll</Right>
```

Comment:
These should not be suppressed.  Instead you should add these to https://github.com/dotnet/runtime/blob/main/src/libraries/System.Security.Permissions/ref/System.Security.Permissions.netframework.cs which should get them in the netframework facade.

Reply:
I don't think these types are in netstandard... Will look into

Reply:
The 5 types were being added to the netstandard build and in the latest commit I removed them along with the suppressions. I think this makes sense. PTAL

Reply:
The 5 types are not in netstandard, but should be in the netstandard version of S.DS since they are implemented in netfx's version of S.DS (not an OOB assembly, and not in netframework's corelib).

Reply:
Added the 5 types to S.DS due reasons above which supports netfx + netstandard (when netstandard used as a shim).